src/styles/blocks/content.css

```
@@ -32,6 +32,69 @@
   border: 0;
 }
 
+.content blockquote {
+  margin-left: 0;
+  position: relative;
+  z-index: 0;
+  overflow: hidden;
+  padding: var(--offset) calc(2 * var(--offset));
+  border-radius: 0 6px 6px 0;
+}
+
+.content blockquote::before {
+  content: '';
+  opacity: calc(0.1 * var(--is-light-theme-on) + 0.4 * var(--is-dark-theme-on));
```

```suggestion
  opacity: calc(
    0.1 * var(--is-light-theme-on) +
    0.4 * var(--is-dark-theme-on)
  );
```

Было бы клёво разворачивать сложные однострочники и в CSS тоже.